sxl_traffic_controller.md

```diff
@@ -638,7 +638,7 @@ Sets traffic situation the controller uses.<br>Requires security code 2
 |----|-------|----|-----|-------|
 |status|setTrafficSituation|boolean|<ul><li>False</li><li>True</li></ul>|False: Controller uses traffic situation according to own programming<br>True: Contoller uses traffic situation according to command|
 |securityCode|setTrafficSituation|string|[text]|Security code 2|
-|trafficsituation|setTrafficSituation|integer|[1-255]|designation of trafficsituation|
+|traficsituation|setTrafficSituation|integer|[1-255]|designation of traficsituation|
```

Comment:
What's the reason we cannot fix the misspelling - because it would require changes to TLC implementations?

Reply:
Because it's a breaking fix.

We know that at least one implementation depends on the misspelling of 'traficsituation'. Fixing it would cause a situation where it only works in updated TLC and systems. Most likley it would take many years before the fix is applied in all TLC's, until then 'traffic situation' may or may not work, depending on if the TLC is up to date or not.